lib/user_agent/browsers/deezer.rb

```diff
@@ -0,0 +1,95 @@
+class UserAgent
+  module Browsers
+    # Deezer/6.2.28.73 (Android; 11; Mobile; fr) samsung SM-G973F
+    # Deezer/6.0.10.201 (Android; 5.1.1; Tablet; nl) samsung SM-T280
+    # Deezer/8.39.0.6 CFNetwork/1220.1 Darwin/20.3.0
+    # Deezer/4.34.10 (Electron; windows/10.0.19041; Desktop; de)
+    # Deezer/4.34.10 (Electron; osx/10.16.0; Desktop; en)
+    # Deezer/4.18.50 (Electron; linux/5.11.0-16-generic; Desktop; us)
+    # Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Deezer/4.25.0 Chrome/73.0.3683.121 Electron/5.0.10 Safari/537.36
+    # Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Deezer/4.27.0 Chrome/73.0.3683.121 Electron/5.0.10 Safari/537.36
+    # Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Deezer/4.27.0 Chrome/76.0.3809.146 Electron/6.1.12 Safari/537.36
+    class Deezer < Base
+      include DesktopClassifiable
+
+      DEEZER        = 'Deezer'
+      LINUX_REGEX   = /linux/.freeze
+      LINUX_SLASH   = 'linux/'
+      OSX_REGEX     = /osx/.freeze
+      OSX_SLASH     = 'osx/'
```

Comment:
Should these belong in the parent class?

Reply:
We have "OSX" in the Base class, this is a custom one with lower case and slash at the end.